GUI/Controls/EditModSearch.cs

```diff
@@ -189,6 +189,12 @@ private void FilterTextBox_KeyDown(object sender, KeyEventArgs e)
             switch (e.KeyCode)
             {
                 case Keys.Enter:
+                    // Bypass the timer for immediate update
+                    e.Handled = true;
```

Comment:
Apparently Windows plays an annoying warning sound when pressing enter in single line text boxes. Adding `e.SuppressKeyPress = true` avoids this. (https://stackoverflow.com/a/16350929)

Reply:
Huh, I must have that sound disabled in my Windows somehow. Still, I see no harm in adding this. Force-pushed.